src/matchmaking-threads.ts

```diff
@@ -0,0 +1,144 @@
+import { MatchmakingThread } from '@/models/matchmakingThreads';
+import { User } from '@/models/users';
+import { sendEventLogMessage } from '@/util';
+import { getDB } from './db';
+import { ChannelType, EmbedBuilder } from 'discord.js';
+import type { Client, Message } from 'discord.js';
+
+export async function handleMatchmakingThreadMessage(message: Message): Promise<void> {
+	const matchmakingChannelId = getDB().get('channels.matchmaking');
```

Comment:
We tend to follow the "effective go" style of variable names, even outside of go, when it comes to things like initialisms and acronyms. Which means variables like `matchmakingChannelId` should be `matchmakingChannelID`

Reply:
I just used this case because discord.js uses it.